### DISCUSSION TITLE
拉取项目要令牌

### DISCUSSION THREAD

**Discussion by githubliunengwen** [zuihou/zuihou-admin-cloud#329] [01.常见问题]
怎么查看令牌

**Comment by zuihou** [comment]
？？？？ 看不懂

**Reply by githubliunengwen** [reply]
项目不是导入吗，我拉不下来
![1708240359424](https://github.com/dromara/lamp-cloud/assets/118824992/5c972e1d-7a48-4ce5-b61b-a865e86f2454)

**Reply by zuihou** [reply]
地址都不对，要导入啥？

**Reply by githubliunengwen** [reply]
这个怎么令牌是啥呀

![recording](https://github.com/dromara/lamp-cloud/assets/118824992/170daef4-aea3-4683-a329-99b25efe6570)

**Reply by zuihou** [reply]
你看的是企业版文档，企业版用户才有这个地址。


开源版的就是这个项目，不看这个文档。
